ubuntu_updates_check.sh

```diff
@@ -23,15 +23,15 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-OUTPUT=$(/usr/lib/update-notifier/apt-check 2>&1)
+OUTPUT=$(apt list --upgradable 2>/dev/null)
 
 if [ $? -ne 0 ]; then
     echo "Failed to retrieve a number of pending updates"
     exit 100
 fi
 
-PENDING_OTHER=$(echo "${OUTPUT}" | cut -d ";" -f 1)
-PENDING_SECURITY=$(echo "${OUTPUT}" | cut -d ";" -f 2)
+PENDING_OTHER=$(echo "${OUTPUT}" | grep -v "Listing..." | grep -v "/$(lsb_release -cs)-security" | wc -l)
+PENDING_SECURITY=$(echo "${OUTPUT}" | grep -v "Listing..." | grep "/$(lsb_release -cs)-security" | wc -l)
```

Comment:
I did a quick test and noticed that some packages on my system are available via both channels, so the pattern here needs to match a leading slash or comma in order to match:

```
ruby1.9.1-dev/trusty-updates,trusty-security 1.9.3.484-2ubuntu1.13 amd64 [upgradable from: 1.9.3.484-2ubuntu1.7]
```